easybuild/easyconfigs/m/mordecai/mordecai-2.0.1-foss-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,133 @@
+easyblock = 'Bundle'
+
+name = 'mordecai'
+version = '2.0.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/openeventdata/mordecai'
+description = """mordecai is a full text geoparsing as a Python library.
+Extract the place names from a piece of text, resolve them to the
+correct place, and return their coordinates and structured geographic
+information."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+exts_download_dep_fail = True
+
+pyver = '3.6.4'
+pymajmin = ''.join(pyver.split('.')[:2])
```

Comment:
@akesandgren Minor cleanup: `pymajmin` isn't used anywhere, so just remove?

And then there's no need for `pyver` anymore either, just specify `'3.6.4'` directly in `dependencies` for `'Python'`?